tools/sonarcloud/README.md

```diff
@@ -17,6 +17,30 @@ The following Pon maintained quality profiles are available:
 
 - [Javascript](https://github.com/pondevelopment/pon-developer-guide/blob/main/tools/sonarcloud/sonarcloud_quality_profile_javascript.xml)
 
+### Code Coverage Reporting
+SonarCloud supports [reporting of test coverage](https://docs.sonarcloud.io/enriching/test-coverage/overview/) information. This enables teams and Management to get more insight into the code coverage of their project.
```

Comment:
Perhaps some additional information about how SonarCloud can be leveraged as the central hub for code quality, especially with the external analyser reports